apps/web/.env.example

```
@@ -1,55 +1,32 @@
+# Required if Google Form webhook endpoint is enabled
+GOOGLE_FORM_WEBHOOK_SECRET=replace-with-random-secret
 
-# ===========================================
-# 포트 설정 (선택)
-# ===========================================
+# Required for local docker compose
 WEB_PORT=3000
 
+# Optional email fallback
+RESEND_API_KEY=re_xxx
+EMAIL_FROM=BinBang <noreply@example.com>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the RESEND_API_KEY key

(UnorderedKey)